voice/methods/calls/getCallsCallId.md

```diff
@@ -0,0 +1,79 @@
+{% method %}
+
+## Retrieve Call Information
+Retrieve the current state of a specific call.
+
+The `state` attribute indicates the current state of the call. Valid values are `created`, `answered` and `disconnected`.
```

Comment:
The states here are different from the states in Catapult (`started` / `rejected` / `active` / `completed` / `transferring`). I chose those based on the names of the callback events we send: `Answer` / `Disconnect`, but these state names can be changed if anyone opposes to them.

Reply:
I've been thinking about changing the `created` state to `Initiated` since we got an `Initiate` callback.